app/repositories/account_repository.go

```diff
@@ -13,3 +17,33 @@ func New(q *pgxpool.Pool) *AccountRepository {
 		q: q,
 	}
 }
+
+func (a AccountRepository) FindAll() ([]entities.Account, error) {
+	rows, err := a.q.Query(context.Background(), "SELECT id, cpf, name, balance, created_at FROM accounts")
+	if err != nil {
```

Comment:
Quando criar os novos métodos de repository, coloca sempre um parâmetro `ctx context.Context`, vc está sempre passando um context hardcoded com `context.Background()`. Não precisa fazer a modificação neste PR aqui, mas o próximo método do repository q vc criar, vc coloca esse parâmetro. Blz?
```suggestion
func (a AccountRepository) FindAll(ctx context.Context) ([]entities.Account, error) {
	rows, err := a.q.Query(ctx, "SELECT id, cpf, name, balance, created_at FROM accounts")
	if err != nil {
```